docs/02-app/01-building-your-application/08-deploying/01-production-checklist.mdx

```diff
@@ -0,0 +1,137 @@
+- [Webpack Bundle Analyzer](https://github.com/vercel/next.js/tree/canary/packages/next-bundle-analyzer) – Creates a visual, interactive map of your webpack output files to show where space is used.
+- [bundlejs](https://bundlejs.com/) - A browser tool for bundling and minifying projects, showing compressed sizes.
+
+Additionally, make sure to implement the recommended optimization strategies.
+
+- Minimize Client-Side JavaScript - It's important to keep JavaScript on the client-side as small as possible for quicker website loading and a better user experience.
+- Leverage [Server Components](https://nextjs.org/docs/app/building-your-application/rendering/server-components) for efficient rendering – Server Components enable rendering on the server, which can significantly reduce the size of your client-side JavaScript bundle.
+- [Fetching data on the server](/docs/app/building-your-application/data-fetching/patterns#fetching-data-on-the-server)
+- [Parallel Data Fetching](https://nextjs.org/learn/dashboard-app/fetching-data#parallel-data-fetching)- Reduce waterfalls and unblock pages by fetching data in parallel.
+
+#### Optimizing with `@next/bundle-analyzer`
+
+[`@next/bundle-analyzer`](https://www.npmjs.com/package/@next/bundle-analyzer) is a plugin for Next.js that helps manage the size of your JavaScript files. It shows you how big each part of your webpack output is. This is helpful because you can find and fix parts of your code that are too large. For example, you might remove large dependencies, split your code, or only load some parts when needed.
+
+How to use `@next/bundle-analyzer`:
+
+1. **Install**: Run `npm install @next/bundle-analyzer webpack-bundle-analyzer`.
+2. **Configure**: Add the bundle analyzer's settings to your `next.config.js`.
+
+```js
+const withBundleAnalyzer = require('@next/bundle-analyzer')({
+  enabled: process.env.ANALYZE === 'true',
+})
+module.exports = withBundleAnalyzer({})
+```
+
+3. **Run Analysis**: Use `ANALYZE=true npm run build` to start looking at your file sizes.
+4. **Review Findings**: Check the report to see where you can make improvements.
+
+Doing this regularly while you develop and before deploying your site can help you identify large bundles earlier.
+
+### Using Lighthouse for performance analysis
+
+[Google Lighthouse](https://developer.chrome.com/docs/lighthouse/overview) is a powerful tool for assessing Core Web Vitals, along with other important aspects such as performance, accessibility, best practices, and SEO. To get the most accurate results:
+
+1. Use a production build of your Next.js application.
+2. Run Lighthouse in incognito mode to prevent browser extensions from affecting the results.
+
+Lighthouse is available directly within Chrome DevTools, as a command-line tool, and online through [Google PageSpeed Insights](https://pagespeed.web.dev).
+
+### Utilizing Vercel's dashboard (for Vercel deployments)
```

Comment:
We might want to instead just link to the Vercel production checklist versus this section 👍